src/plugins/data/common/search/tabify/tabify.test.ts

```diff
@@ -214,6 +215,7 @@ describe('tabifyAggResponse Integration', () => {
       // check for something like an average bytes result
       function expectAvgBytes(val: string | number) {
         expect(typeof val).toBe('number');
+        assert(typeof val === 'number');
```

Comment:
Curious, is this intentional? I don't think we need the extra `assert` here.

Reply:
This was from me, I was trying to use [Typescripts `assert` function](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions) as a type guard. I removed it in 7c76dd386e5fe927ed89619b66618571595f2178.